static/js/settings_users.js

```diff
@@ -667,14 +674,19 @@ function handle_bot_form(tbody, status_field) {
             // Note: Rendering this is quite expensive in
             // organizations with 10Ks of users.
             owner_widget = new DropdownListWidget(opts);
+
+            $("#bot-role-select").val(bot.role);
+            $("#bot-role-select")
+                .find(`option[value="${CSS.escape(settings_config.user_role_values.owner.code)}"]`)
+                .hide();
```

Comment:
Have we decided to not allow a bot with role `Owner`? I don't think it is mentioned in the issue but we can confirm. And if so, we should make sure we add the check in API too.

Reply:
It wasn't mentioned in the issue, but I just thought why will a bot get the `Owner` role. 
If that's not the case, shall I allow the `Owner` role for bots?

Reply:
I am not sure too, but for various purposes we check the permission of its owner. @alya thoughts?